ibu-imager/internal/ops/execute.go

```diff
@@ -33,5 +33,5 @@ func (e *executor) Execute(command string, args ...string) (string, error) {
 	cmd.Stdout = &stdoutBytes
 	cmd.Stderr = &stderrBytes
 	err := cmd.Run()
-	return strings.TrimSpace(stdoutBytes.String()), errors.Wrap(err, stderrBytes.String())
```

Comment:
Unlike `errors.Wrap`, `fmt.Errorf` will not return `nil` if the only instance of `%w` in its string is nil